app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -0,0 +1,43 @@
+class MailgunWebhooksController < ApplicationController
+  skip_before_action :redirect_to_getyourrefund
+  skip_before_action :set_visitor_id
+  skip_before_action :set_source
+  skip_before_action :set_referrer
+  skip_before_action :set_utm_state
+  skip_before_action :set_sentry_context
+  skip_before_action :check_maintenance_mode
+  skip_after_action :track_page_view
+  skip_before_action :verify_authenticity_token
+  before_action :valid_mailgun_post?
+
+  def create_incoming_email
+    sender_email = params["sender"]
```

Comment:
Is this the `Sender: ` email header? Or is it something computed by Mailgun? Or is it the "Envelope From"?

I think the "From:" header is a better here, but happy to discuss.

Reply:
Non-blocking!

Reply:
They document it here: https://documentation.mailgun.com/en/latest/user_manual.html#routes

> sender of the message as reported by `MAIL FROM` during SMTP chat. Note: this value may differ from `From MIME header.

Reply:
decided, we'll go with sender for now

Reply:
comment with docs is nice